src/helpers/routeHelper.js

```diff
@@ -0,0 +1,47 @@
+import config from '../config/config.json';
+import { ProductController, ReviewController, UserController } from '../controllers';
+
+export class ProductRouteUtils {
+  static addProduct(req, res) {
+    const product = ProductController.addProduct(req.body.name, req.body.brand, req.body.price);
```

Comment:
Maybe it's ok but at the moment it's possible to add a product with all empty fields. I'd add some validation.

Reply:
Done it